pkg/kncloudevents/good_client.go

```diff
@@ -62,6 +64,18 @@ func NewDefaultClient(target ...string) (cloudevents.Client, error) {
 	return NewDefaultClientGivenHttpTransport(t, nil)
 }
 
+func NewHTTPClient(target ...string) (*http2.Protocol, error) {
```

Comment:
Why are you using the protocol? I think you should use directly the net/http.Client here

Reply:
One with some pre-configuration is provided in kncloudevents module

Reply:
thanks for the pointer, was not sure what I was supposed to use here, will switch to net/http.Client

Reply:
I renamed this to newHTTPProtocol,
perhaps I'm misunderstanding something. I'll ping you on slack to clarify doubts.

Reply:
removed.